server.js

```diff
@@ -1865,8 +1865,11 @@ if (config.startServer) {
         };
         function idleErrorHandler(err) {
           logger.error('idle client error', err);
-          // https://github.com/PrairieLearn/PrairieLearn/issues/2396
-          process.exit(1);
+          Sentry.captureException(err);
+          Sentry.close().finally(() => {
+            // https://github.com/PrairieLearn/PrairieLearn/issues/2396
```

Comment:
Looks like this issue is no longer relevant?

Reply:
True! Let me remove this.